lib/cli/generators/MARKO/template/.storybook/addons.js

```diff
@@ -1,2 +1,2 @@
 import '@storybook/addon-actions/register';
-import '@storybook/addon-links/register';
+import '@storybook/addon-knobs/register';
```

Comment:
You need to add it as dependency in `MARKO/index.js` then

Reply:
Done 👍

Reply:
Please also commit the new files in `snapshots/marko/components` (`git add -A` should help)

Reply:
Yes, snapshots files are part of PR.